src/app/_components/feature/index.jsx

```diff
@@ -0,0 +1,47 @@
+import styles from './index.module.scss'
+
+export const Feature = ({ className = '' }) => {
+  return (
+    <section className={`${styles['inner']} ${className}`}>
```

Comment:
[must]
innerはglobal.scssに定義されているものを使いたいです！

https://github.com/gizumo-education/front-coding/blob/6c23a77398236235cf5eb3b7761a77854c497adb/src/styles/_globals.scss#L69-L78

現状の書き方だと、stylesのinnerを参照しているので、/feature/index.module.scssのinnerが使用されてしまいます...
これだと全てのコンポーネントで同じスタイルのinnerを定義する必要があるため、保守性の面でも改善したい項目になります。

global.scssはlayout.jsxでimportされているため、どのページからでも参照できるようになっています。
そのため、

```suggestion
    <section className='inner'>
```

のように定義するとglobal.scssのinnerを適用することができます！
この方法にすればコンポーネント毎にinnerを定義する必要がなくなるため、修正をお願いします！